about.html

```diff
@@ -0,0 +1,122 @@
+              <p class ="text-2">please contact us trough email for further questions and how to make your software applicable </p>
+              <a href="#" class="link">metrixtech3@gmail.com</a>
+
+        </section>
+        <section class="conatiner">
+            <div class="our-history">
+                <h3  class="title-3">Our History</h3>
+                <p class ="text-2">We got the idea to found Metro Labs in 2018—inspired by African union’ absurdly eclectic start-up ecosystem. Back then, we were working from a cold brew-filled co-working space. There, we began meeting brilliant innovators who had no way of building their digital projects. With our backgrounds in product and software, we realized we were uniquely equipped to help founders crystallize their ideas into incredible products—so we did. And we still do. Now, with a growing team of +30 and decades of development experience between us, we’ve proudly deployed hundreds of products and expanded to working with everyone from worldwide nonprofits to mature companies. And, while our Goji Methodology varies between our diverse clients, its principles remain simple: Build user-centered products.
+                     Build beautiful products. And build them right—the first time </p>
+                <div class="history-logo">
+                     <img src="assets/images/logo.jpg" alt="our first logo">
+                     <div>
+                        <h3 class="text-1">Upper Technologies</h3>
+                        <p class="text-3">Started: OCT 2015</p>
+                     </div>
+                 
+                </div>
+    
+            </div>
+            <div class="our-experiance">
+                <h3 class="title-3">Our Success Story</h3>
+                <p class="text-3">We complete many projects succesfuly for the last couple of years</p>
+                <div class="experiances">
+                    <article class="success">
+                    <img src="/assets/images/course-material-access.png" alt=" online course ware image" class="experiance-img">
+                    <div class="experiance-text">
+                        <h3 class="expriance-title"> Course Ware System </h3>
+                        <p class="text-3"> Online Course Ware System is a user-friendly and interactive e-learning system  </p>
+                    </div>
+                    </article>
+                    <article class="success">
+                   <img src="/assets/images/online-inventory.png" alt="online inventory system" class="experiance-img">
+                          
+                   <div class="experiance-text">
+                   <h3 class="expriance-title">  Inventory Management System</h3>
+                   <p class="text-3">automate inventory management process by implementing online inventory management  </p>
+                    </div>
+
+                        </article>
```

Comment:
- [ ] in the mobile version your images are not aligned correctly and one is bigger than the other so please try to fix that in order for your website to be responsive on all screens
![image](https://user-images.githubusercontent.com/61790587/169376246-c3ea02e4-70e0-4591-ab98-f5b91e9ed9e7.png)